CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [Unreleased]
 
+
+## [1.10.1] - 2016-07-02
```

Comment:
I was thinking 1.10.1 because there are no new features, only difference is (officially declared) compatibility with ESLint 3.